samples/zello_world/zello_world.cpp

```diff
@@ -61,13 +62,22 @@ int main( int argc, char *argv[] )
     {
         tracing_runtime_enabled = true;
     }
+    if( argparse( argc, argv, "-legacy", "--enable_legacy_init" ) )
```

Comment:
minor suggestion, name -legacy to -legacy_init as many other future options may be legacy in nature.

Reply:
fixed